### ISSUE TITLE
add ts-rest

### ISSUE THREAD

**Issue by northamerican** [bogeeee/RPCFrameworksComparison#1]
might qualify for this list. it's labelled as "rpc-like". 
https://github.com/ts-rest/ts-rest

**Comment by bogeeee** [comment]
Yes, it is. I won't have time for it, but maybe someone, who's familiar with it wants to do a PR ?
